src/views/AddStation.js

```diff
@@ -0,0 +1,186 @@
+import React, { useState, Fragment } from "react";
+import axios from "axios";
+import { API_ROOT, toolTipCopy } from "../app/app-helpers";
+import { useAuthState } from "../auth/auth-context";
+import { QuestionMarkToolTip } from "../app/app-helpers";
+import { checkJwt } from "../auth/auth-helpers";
+import Spinner from "../app/components/Spinner";
+import CircleCheck from "../assets/CircleCheck.svg";
+
+export default function AddStation() {
+  const { jwt } = useAuthState();
+  // form state
+  const [stationName, setStationName] = useState(``);
+  const [latitude, setLatitude] = useState(``);
+  const [longitude, setlongitude] = useState(``);
+  const [altitude, setAltitude] = useState(``);
```

Comment:
Needs changed to Elevation - to be consistent. check in with Mike r.e. the messaging and the tooltip